src/geos_types.jl

```diff
@@ -309,6 +309,63 @@ const geomtypes = [
     GeometryCollection,
 ]
 
+function Base.:(==)(geo1::AbstractGeometry, geo2::AbstractGeometry)::Bool
+    compare(==, geo1, geo2)
+end
+function Base.isequal(geo1::AbstractGeometry, geo2::AbstractGeometry)::Bool
+    compare(isequal, geo1, geo2)
+end
+function compare(≅, geo1::AbstractGeometry, geo2::AbstractGeometry)::Bool
```

Comment:
Do you want to also define `Base.isapprox`? Seems like it's all set up for it.

Reply:
Done